app/views/handlers/list_add_question.py

```diff
@@ -13,3 +15,14 @@ def handle_post(self):
         )
         self.questionnaire_store_updater.update_answers(self.form.data, list_item_id)
         return super().handle_post()
+
+    def _resolve_custom_page_title_vars(self) -> MutableMapping:
+        # For list add blocks, no list item id is yet available. Any custom `page_title`
+        # `list_item_position` string formatter is instead resolved to the length of
+        # the list identified by the `list_name` attribute of the current Location
+        # object.
```

Comment:
Some of this comment is redundant (it's clear from the code how we get the current list name), and some is incorrect (it should be the list length plus one). Maybe something like:
```suggestion
        # For list add blocks, no list item id is yet available. Instead, we resolve 
        # `list_item_position` to the position in the list it would be if added.
```